### PR TITLE
Add machine readable output (-o flag) to kn source apiserver describe

### LINKED ISSUE #1096: Add machine readable output (-o flag) to kn source apiserver describe
use case: `kn source apiserver describe NAME -oyaml`

see `kn service describe NAME -oyaml` for ref

/kind good-first-issue

### PR DESCRIPTION
**Pull request by arghya88**
Signed-off-by: Arghya Sadhu <arghya88@gmail.com>

## Changes

*Add machine readable output (-o flag) to kn source apiserver describe

## Reference

Fixes #1096

### CODE REVIEW

File: pkg/kn/commands/source/apiserver/describe.go [hunk 1]
```diff
@@ -18,23 +18,34 @@ import (
...
-		Example: `
-  # Describe an ApiServer source with name 'k8sevents'
-  kn source apiserver describe k8sevents`,
+
+	// For machine readable output
+	machineReadablePrintFlags := genericclioptions.NewPrintFlags("")
+
+	command := &cobra.Command{
+		Use:     "describe NAME",
+		Short:   "Show details of an api-server source",
```
Comment (navidshaikh) [inline pkg/kn/commands/source/apiserver/describe.go:47]: the way we word `api server` in `Short` and `Example` can be aligned
I'd go for all lower case, `api-server` lgtm. See `kn service describe -h` for example.

File: pkg/kn/commands/source/apiserver/describe.go [hunk 2]
```diff
@@ -18,23 +18,34 @@ import (
...
 	"knative.dev/client/pkg/kn/commands"
 	"knative.dev/client/pkg/printers"
 )
 
+var describeExample = `
+  # Describe an ApiServer source with name 'k8sevents'
+  kn source apiserver describe k8sevents
+
+  # Describe an ApiServer source with name 'k8sevents' in YAML format
+  kn source apiserver describe k8sevents -o yaml`
```
Comment (navidshaikh) [inline pkg/kn/commands/source/apiserver/describe.go:37]:
```suggestion
  # Describe an api-server source with name 'k8sevents'
  kn source apiserver describe k8sevents

  # Describe an api-server source with name 'k8sevents' in YAML format
  kn source apiserver describe k8sevents -o yaml`
```

Comment (navidshaikh) [reply pkg/kn/commands/source/apiserver/describe.go:37]: (make sure to run `./hack/build.sh` after these changes so same could be reflected in .md docs)

**Review by navidshaikh — CHANGES_REQUESTED** [review]
just a minor nit to align the wording in example/usage, beside lgtm!
thank you for bringing these features!
cc: @joshuawilson

File: pkg/kn/commands/source/apiserver/describe.go (hunk 2 above)
Comment (arghya88) [reply pkg/kn/commands/source/apiserver/describe.go]: Done

**Review by navidshaikh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**